GSA/digitalgov-pra
# Remove duplicate ID on main element

Resolves trello card `PRA a11y: Ensure landmarks are unique` by removing duplicate ID on child of main element.

I've also updated minor and patch dependencies (non-breaking).

# How to test

1. Go to homepage
2. Run axe dev tools
3. There should be **no** a11y error
